src/index.ts

```diff
@@ -327,6 +330,53 @@ app.get('/api/advertising-info', (_req, res) => {
 
 app.use('/api/slise-ad-rules', sliseRulesRouter);
 
+app.post('/api/magic-square-quest/start', async (req, res) => {
```

Comment:
Please consider moving these entrypoints into a separate router because this file is already bloated